README.md

```diff
@@ -2,32 +2,27 @@
 
 # FastBoot Website
 
-This is the Ember.js app that powers the FastBoot website,
-<https://ember-fastboot.com>.
+This is the Ember.js app that powers the FastBoot website, <https://ember-fastboot.com>.
 
-Most of the content is authored in Markdown and can be found in the
-`markdown` directory.
+Most of the content is authored in Markdown and can be found in the `markdown` directory.
 
 ## Contributing
 
-You don't need this repository to use FastBoot. However, if you'd like
-to contribute documentation or correct errors, you can submit a pull
-request.
+You don't need this repository to use FastBoot. However, if you'd like to contribute documentation or correct errors, you can submit a pull request.
 
 To run the website locally:
 
-* `git clone https://github.com/ember-fastboot/fastboot-website`
-* `cd fastboot-website`
-* `npm install`
-* `bower install`
-* `ember server`
-* Visit the app at [http://localhost:4200](http://localhost:4200).
+- `git clone https://github.com/ember-fastboot/fastboot-website`
+- `cd fastboot-website`
+- `npm install`
+- `bower install`
+- `ember fastboot --serve-assets`
```

Comment:
@arjansingh I think we're going to be able to integrate into the built-in `ember serve` (based on the work @stefanpenner is doing), so probably best to go with that since that will be the FastBoot option soon anyway. :)